### PR TITLE
UI: Fix icons alignment in header

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
The icons seemed a bit off, so added a touch of CSS.
Subtle but irritating

Before:
Screenshot from 2022-08-31 13-10-24

Now:
Screenshot from 2022-08-31 13-10-13

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Oh, yes, hmm. The feeling you get here is because of the badge on the K8s logo. The thing is that the logos should all be the same size, while the badge is extra and as such might hang past other logos without badges.  

Do this, if you would: look at other systems (phones, macOS, others) that have badges and compare. Here's a screenshot from my system:
Screen Shot 2022-08-31 at 7 34 04 AM

**Comment by Nikhil-Ladha** [comment]
Hmm....makes sense.
Though there was a slight issue with the overall icons set and for that the `align-items: center` is still needed.

File: ui/components/Header.js [hunk 1]
```diff
@@ -289,7 +291,7 @@ function K8sContextMenu({
           ? 'menu-list-grow'
           : undefined}
         aria-haspopup="true"
-        style={{ marginRight : "0.5rem" }}
+        style={{ marginRight : "0.5rem", top : "0.25rem" }}
```
Comment (Nikhil-Ladha) [inline ui/components/Header.js:294]:
```suggestion
        style={{ marginRight : "0.5rem" }}
```

**Review by leecalcote — APPROVED** [review]
LGTM 👍

**Pull request merged**